docs/2.deploy/20.providers/deno-deploy.md

```diff
@@ -25,11 +25,11 @@ deployctl deploy --project=my-project server/index.ts
 
 You just need to include the deployctl GitHub Action as a step in your workflow.
 
-You do not need to set up any secrets for this to work. You do need to link your GitHub repository to your Deno Deploy project and choose the "GitHub Actions" deployment mode. You can do this in your project settings on https://dash.deno.com.
+You do not need to set up any secrets for this to work. You do need to link your GitHub repository to your Deno Deploy project and choose the "GitHub Actions" deployment mode. You can do this in your project settings on [here](https://dash.deno.com)
```

Comment:
Do not use `here` inside a link to avoid accessibility issue.

See https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#:~:text=%E2%80%9CHere%E2%80%9D%20Conceals%20What%20Users%20Are,won't%20have%20a%20clue.